### ISSUE TITLE
Update table headings on Veterans disability compensation rates tables with more descriptive language

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#46043]
## Issue Description
We recently learned there were several 508 accessibility issues on our [2022 Veterans disability compensation rates](https://www.va.gov/disability/compensation-rates/veteran-rates/) page. The issue that our team needs to fix is to make the headings for each table more descriptive so that a screen reader knows what the table is showing. 

For each of the tables labeled "Basic rates for monthly payments" and "Added amounts," we need to change it to the following (though update the percentage amounts for each table):

Basic monthly rates for 30% to 60% disability rating
Added amounts for 30% to 60% disability rating

We should also make the same changes on these past rates pages

## Tasks
- [ ] Update table headings on the current rates page and each of the past rates pages (Aliyah)
- [ ] Copyedit (Megan)
- [ ] Publish pages (Megan)

## Acceptance Criteria
- [ ] Live pages have updated headings

**Comment by aliyahblackmore** [comment]
Hi @megzehn this is ready for you! The 2022 and past rates pages have been updated!

**Some notes/flag:**

1. Throughout the page (in text before the accordions and on the end of the page) "**Basic rates table**" has been changed to "**Basic monthly rates table**"  Laura gave the ok to make this change!

2. On the [2019](https://www.va.gov/disability/compensation-rates/veteran-rates/past-rates-2019/) past rates page, the accordions in "With dependents, including children" all include "—all rates in $ U.S. dollars" in the header. I left this as is for the time being, but wondering if this should be moved to the table, similar to the tables in the accordions in "With a dependent spouse or parent, but no children?"

2022 
Preview link:  http://preview-prod.vfs.va.gov/preview?nodeId=986

2021
Preview link: http://preview-prod.vfs.va.gov/preview?nodeId=39312

2020 
Preview link: http://preview-prod.vfs.va.gov/preview?nodeId=11075

2019

Preview link: http://preview-prod.vfs.va.gov/preview?nodeId=4714

**Comment by megzehn** [comment]
All 4 pages have been reviewed and published. Thanks, everyone! 

Over to you @RLHecht

**Comment by RLHecht** [comment]
These pages are live. Closing ticket.
